public/lesson/lesson.html

```diff
@@ -0,0 +1,10 @@
+<div class='panel panel-info'>
+  <div class='panel-heading'>
+    <h3>{{lesson.title}} - Learning Objectives</h3>
+  </div>
+  <div class='panel-body'>
+    <learning-objective ng-repeat="tag in tags" expanded id="{{tag.content}}"></learning-objective>
+    <h1 ng-show="!tags.length">No Learning objectives set</h1>
```

Comment:
The one thing I don't like here is that "No Learning objectives set" shows up and then quickly disappears when there are tags.  There must be a better way?